lib/presentation/home/captive_screen.dart

```diff
@@ -27,19 +30,41 @@ class CaptivePage extends StatelessWidget {
                   child: Padding(
                     padding: const EdgeInsets.all(8.0),
                     child: Column(
-                      children: const [
-                        // TODO: Leaving this commented for now, not sure what it should do ...
-                        // Row(
-                        //   mainAxisAlignment: MainAxisAlignment.end,
-                        //   children: [
-                        //     CustomFAB(
-                        //       isMini: true,
-                        //       onPressed: () {},
-                        //       child: const Icon(Icons.settings),
-                        //     )
-                        //   ],
-                        // ),
-                        ContentPlaceholder(
```

Comment:
I guess this was supposed to open the settings, according to the design of the captive page, there was a 'settings' mini-fab in the top right corner.

Reply:
Yes, but this did not match the design, so I copied the button from the profile page.
Is it not being displayed for you?

Reply:
Yeah, making some last minute changes and fixing some exceptions from captive screen, will make a commit in a few minutes.